lib/network/modules/components/shared/ComponentUtil.js

```diff
@@ -116,13 +116,13 @@ class ComponentUtil {
   }
 
   /**
-   * Check if given value is acceptable as an integer.
+   * Check if given value is acceptable as an number.
    *
    * @param {*} number value to check; can be anything at this point
-   * @returns {boolean} true if valid integer value, false otherwise
+   * @returns {boolean} true if valid number value, false otherwise
    */
-  static isValidInteger(number) {
-    return (typeof number === 'number' && Number.isInteger(number) && Number.isFinite(number));
+  static isValidNumber(number) {
+    return (typeof number === 'number' && Number.isFinite(number));
```

Comment:
Doesn't `isValidNumber(x) === Number.isFinite(x)` for any `x`?

Anyway “acceptable as an number” [sic] is very vague.

Reply:
As I see it will isFinite not let through any numbers who not are acceptable as a finite number.
Maybe it's to weak validation, have you any suggestions or ideas?

Reply:
I don't see a reason why it would be too weak. Personally I would just ditch `isValidInteger` and use `Number.isFinite` directly.